src/brsTypes/components/RoSGNode.ts

```diff
@@ -1073,6 +1070,19 @@ export class RoSGNode extends BrsComponent implements BrsValue, BrsIterable {
             return new BrsString(this.type);
         },
     });
+
+    protected registerFields(fields: FieldModel[]) {
+        if (!fields) {
```

Comment:
I thought empty arrays are truthy in javascript?  If so, this will never be entered.  

I also don't think it's necessary since the forEach is a noop in this case (see RoSGNode constructor as a reference).  Empty arrays are forEach safe.

Reply:
I need to disconnect my brain from a land where we've got interop between CoffeeScript & TypeScript. 😄 My intent here was to guard against `null` being supplied by a caller, but that's unnecessary since we're fully within the realm of TS here.